helm-app-operator/pkg/helm/controller/reconcile.go

```diff
@@ -28,18 +28,51 @@ type helmOperatorReconciler struct {
 	mutex                sync.RWMutex
 }
 
+const (
+	finalizer = "uninstall-helm-release"
```

Comment:
Is there a format requirement or convention for finalizer names?

Reply:
Not that I'm aware of. Finalizers can be arbitrary strings. And this one seems fine since its descriptive enough.
https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers